up-core-api/uprotocol/upayload.proto

```diff
@@ -1,25 +1,16 @@
 /*
  * Copyright (c) 2023 General Motors GTO LLC
```

Comment:
proto updates seem to be messed up and still include the GM copyright notice...

Reply:
never copy & paste ...